### PR TITLE
Cb 24 - Register shooter with competition

### PR DESCRIPTION
**Pull request by lrobert142**

### CODE REVIEW

File: resources/sql/entries.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-- :name entries-create! :<! :1
+-- :doc Creates a new entry
+INSERT INTO entries (id, side, grade)
```
Comment (axrs) [inline resources/sql/entries.sql]: id should be auto generated
`side` should be `sid`

Comment (axrs) [reply resources/sql/entries.sql]: `id` should be `competition_id`

File: resources/sql/entries.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+VALUES (:id, :sid, :grade);
```
Comment (axrs) [inline resources/sql/entries.sql]: `:id -> :competition-id::UUID`

**Review by axrs — CHANGES_REQUESTED** [review]

**Review by CalebMacdonaldBlack — APPROVED** [review]

File: src/clj/centrebull/db/entries.clj [hunk 3]
```diff
@@ -0,0 +1,19 @@
+(ns centrebull.db.entries
+  (:require [clojure.string :refer [split lower-case]]
+            [clojure.set :refer [map-invert]]
+            [centrebull.db.util :refer [mapper]]
+            [centrebull.db.core :refer [entries-create!]]))
+
+(def ^:private key-map {:competition/id          :id
```
Comment (axrs) [inline src/clj/centrebull/db/entries.clj]: change to competition-id
add :entry/id :id

**Review by axrs — APPROVED** [review]

**Pull request merged**